src/problem3.rb

```diff
@@ -0,0 +1,23 @@
+
+# Given an array of numbers having size N
+# in which every number is between 1 and N,
+# determine if there are any duplicates in it.
+# You are allowed to destroy the array if you like.
+# Do not sort the array or use bit vectors.
+# Try to work within the array without using any
+# other temporary data structures.
+# Do not use 2 nesting for loops since the time
+# complexity would be high.
+# Try to find a solution by executing just one loop.
+class Problem3
+  # @param [Array] numbers
+  # @return [boolean]
+  def self.duplicates?(numbers)
```

Comment:
Really liked the fact that you used `?` in the method name, that is a cool ruby naming convention.